scripts/generate_grapheme_properties.cr

```diff
@@ -0,0 +1,81 @@
+# This script generates the file src/string/grapheme/properties.cr
```

Comment:
Would it make sense to merge this script with the other `generate_unicode_data` script so that there's only 1 thing that generates all unicode data?

Reply:
I think it makes sense to separate these concerns into individual files. They work with different data. The code is more readable when it's clearly separated.

Instead, I would consider adding a Makefile for generating data. That allows us to easily re-generate data with a single command and manage configuration such as Unicode versions in a single place.

Reply:
👍  Works for me. We also have that `Unicode::VERSION` constant we could use at least versus a make env var.